client/src/components/RelatedProducts/Individualcard.jsx

```diff
@@ -0,0 +1,44 @@
+import React, { useContext } from 'react';
+import styled from 'styled-components';
+import sampleData from '../../sampleData';
+
+// should retrieve all related products from initial product page
+// map each individual card with all the information
+
+export default function Individualcard() {
+  return (
```

Comment:
Recommend being consistent with the order of code blocks for readability. Some of the files has return before style components some has return after.  It's all personal preferences and doesn't impact code performance, just my 2 cents.